docs/reference/server-worker-configuration.md

```diff
@@ -89,8 +94,8 @@ The platform name can be one of `windows`, `macos` or `linux`. Each of these can
 | `arguments` | Required | A list of arguments to provide to the base command. The arguments can contain placeholder strings, like `{IMPROBABLE_RECEPTIONIST_HOST}`, which are interpolated by SpatialOS when the worker is launched. For a full list of supported placeholder strings, see our documentation on [worker launch configurations](https://docs.improbable.io/reference/latest/shared/worker-configuration/launch-configuration). |
 
 #### Cloud deployment configuration
-The cloud deployment configuration specifies how to launch the worker in the cloud. It's the same as the local deployment configuration at the moment, with the following exceptions:
+The cloud deployment configuration specifies how to launch a server-worker in the cloud. It's the same as the local deployment configuration at the moment, with the following exceptions:
 - A zip file containing the directory specified in `localBuildDirectory` will be uploaded to the cloud.
 - We'll add additional fields in the future, allowing you to specify where the worker binaries are uploaded.
 
-Note: It rarely makes sense to specify a cloud deployment configuration for any platform other than `linux`, since workers are run in an Linux environment in the cloud.
+Note: We only support the `linux` platform for cloud deployment configurations because workers are always run in an Linux environment in the cloud.
```

Comment:
```suggestion
Note: We only support the `linux` platform for cloud deployment configurations because worker instances are always run in an Linux environment in the cloud.
```

Reply:
Done.